tests/app/views/contexts/test_submit_context.py

```diff
@@ -0,0 +1,165 @@
+from unittest.mock import Mock, patch
+
+from app.data_models.answer_store import AnswerStore
+from app.data_models.list_store import ListStore
+from app.data_models.progress_store import ProgressStore
+from app.utilities.schema import load_schema_from_name
+from app.views.contexts import SubmitQuestionnaireContext
+from tests.app.app_context_test_case import AppContextTestCase
+
+
+class TestStandardSummaryContext(AppContextTestCase):
```

Comment:
I don't think the split of classes and tests is right here. The `SubmitQuestionnaireContext` class sets the submission guidance regardless of whether a summary is included or not. I think we should have a `TestSubmitContext` class that tests for what is common, and a `TestSubmitContextWithSummary` class to test the summary.

Reply:
Have cleaned up the classes to with clear separation of Submit with and without summary.